test/metatx/ERC2771Forwarder.t.sol

```diff
@@ -93,6 +96,68 @@ contract ERC2771ForwarderTest is Test {
             });
     }
 
+    function _tamperedValues(
+        uint8 tamper,
+        ERC2771Forwarder.ForwardRequestData memory request
+    ) private returns (ERC2771Forwarder.ForwardRequestData memory) {
+        return
+            ERC2771Forwarder.ForwardRequestData({
+                from: tamper == 0 ? vm.randomAddress() : request.from,
+                to: tamper == 1 ? vm.randomAddress() : request.to,
+                value: tamper == 2 ? vm.randomUint() : request.value,
+                gas: request.gas,
+                deadline: request.deadline,
+                data: tamper == 3 ? vm.randomBytes(4) : request.data,
+                signature: tamper == 4 ? vm.randomBytes(65) : request.signature
```

Comment:
I guess we can define the `tamper` uint8 as an enum. This way I think the `bound` wouldn't be needed if the arguments are enums too.

Reply:
not possible currently https://github.com/foundry-rs/foundry/issues/6623

Reply:
I'd use an enum internally (as argument for `_tamperedValues` and `_tamperedExecute`)

Any in the public function we can use `TamperType(uint8(bound(tamper, 0, 4)))` to create that enum object